Example/Source/Examples/Interactive/InsertBrickViewController.swift

```diff
@@ -49,47 +67,38 @@ class InsertBrickViewController: BrickApp.BaseBrickController, HasTitle {
         self.navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(InsertBrickViewController.insertBrick))
 
         let section = BrickSection(Section, bricks: [
-            SegmentHeaderBrick(dataSource: self, delegate: self),
-            LabelBrick(BrickIdentifiers.repeatLabel, backgroundColor: UIColor.lightGray, dataSource: self)
+            BrickSection(bricks: [SegmentHeaderBrick("AppearSegmentHeaderBrick", dataSource: self, delegate: self),
+                                  SegmentHeaderBrick("InsertSegmentHeaderBrick", dataSource: self, delegate: self)]),
+            LabelBrick(BrickIdentifiers.repeatLabel, height: .fixed(size: 37), backgroundColor: UIColor.lightGray, dataSource: self)
             ], inset: 10, edgeInsets: UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5))
         section.repeatCountDataSource = self
 
         self.setSection(section)
     }
 
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
-        self.scrollToBottom()
-    }
-
     @objc func insertBrick() {
-        numberOfLabels += 1
-        updateRepeatCounts()
+        let newElement = "BRICK \(data.count + 1)"
+        let index = insertToIndex
+        if selectedInsertSegmentIndex == 2 {
+            data.append(newElement)
+        } else {
+            data.insert(newElement, at: index)
+        }
+        updateRepeatCounts(fixedInsertedIndexPaths: [IndexPath(item: index + 1, section: 1)])
```

Comment:
BrickKit is designed so the consumer doesn't need to know about indexPaths, so this concerns me a bit. I probably want the API to be a bit more in the trend of
```
insertItem(for identifier: String, atIndex index: Int)
```